internal/core/application/operator_service.go

```diff
@@ -836,6 +837,61 @@ func (o *operatorService) FeeAccountBalance(ctx context.Context) (
 	return int64(baseAssetAmount), nil
 }
 
+// ReloadUtxos triggers reloading of unspents for stored addresses from blockchain
+func (o *operatorService) ReloadUtxos(ctx context.Context) error {
+	//get all addresses
+	vault, err := o.vaultRepository.GetOrCreateVault(
+		ctx,
+		nil,
+		"",
+		nil,
+	)
+	if err != nil {
+		return err
+	}
+
+	addressesInfo := vault.AllDerivedAddressesInfo()
+	for _, v := range addressesInfo {
```

Comment:
in #312 a new function fetchUnspents() has been added that already that this work. If we wait for that PR to be merged, this portion of code could be simplified as a simple function call.

Reply:
fetchUnspents lives in the wallet_service isn't it? should we export it or expose isnt'

Reply:
it is a private function of the `application` package, therefore the operator service can use it since it leaves in the same package.

Reply:
you right nvm

Reply:
I woudl either ask @sekulicd to cherrypick or copy paste that method, since your PR is bigger and will go inside provably next week after a bit of more QA

Reply:
Done